### ISSUE TITLE
Crashes out if a file contains UTF-16 surrogates

### ISSUE THREAD

**Issue by kode54** [Ellerhold/fs2es-indexer#22]
It crashes if a UTF-16 encoded file contains surrogates:

```
UnicodeEncodeError: 'utf-8' codec can't encode character '\udcf6' in position 86: surrogates not allowed
```

It should support these, since Windows created UTF-16 files will contain them.

**Comment by kode54** [comment]
Meh, this looks to be an issue with python-elasticsearch.

**Comment by kode54** [comment]
Disregard that, that was an issue that python-elasticsearch had, but this issue is with line 50 of Fs2EsIndexer.py, where it does path.encode('utf-8')... Apparently, I have surrogates in my paths somewhere.

**Comment by MatthiasKuehneEllerhold** [comment]
Fixed in https://github.com/Ellerhold/fs2es-indexer/pull/23, published as 0.4.2

**Comment by MatthiasKuehneEllerhold** [comment]
Thanks for your report and the PR!
